.github/workflows/resync-plan.yaml

```diff
@@ -0,0 +1,46 @@
+name: Resync Plan "v1"
+
+on:
+  pull_request:
+    branches:
+      - main
```

Comment:
This means when opening a pull request for merging into main? Will it also run with new commits pushed to the branch?

Reply:
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request